cmd/controller/controller.go

```diff
@@ -103,11 +111,18 @@ func main() {
 		pvcInformer.Informer(),
 		podInformer.Informer(),
 		importerImage,
-		pullPolicy,
-		verbose)
+		PullPolicy,
+		Verbose)
+	
+	cloneController := controller.NewCloneController(client,
+		pvcInformer.Informer(),
+		podInformer.Informer(),
+		clonerImage,
+		PullPolicy,
```

Comment:
are PullPolicy and Verbose exported fields in the controller? Maybe needed by tests?